PACKAGE-LICENSING

```diff
@@ -1,14 +1,14 @@
-All files in DNF are distributed as GPLv2+ with the exceptions below:
+All files in DNF are distributed as GPL-2.0-or-later with the
+exceptions below:
 
```

Comment:
Is this change correct? Shouldn't `dnf/rpm/transaction.py` remain GPLv1.0? Or are we able to change it to GPLv2+?